tests/tracking/test_utils.py

```diff
@@ -7,11 +8,18 @@
 from mlflow.store.dbmodels.db_types import DATABASE_ENGINES
 from mlflow.store.file_store import FileStore
 from mlflow.store.rest_store import RestStore
-from mlflow.store.sqlalchemy_store import SqlAlchemyStore
+
 from mlflow.tracking.utils import _get_store, _TRACKING_URI_ENV_VAR, _TRACKING_USERNAME_ENV_VAR, \
     _TRACKING_PASSWORD_ENV_VAR, _TRACKING_TOKEN_ENV_VAR, _TRACKING_INSECURE_TLS_ENV_VAR, \
     get_db_profile_from_uri, _download_artifact_from_uri, TrackingStoreRegistry
 
+test_logger = logging.getLogger(__name__)
+
+try:
```

Comment:
One question, is this change necessary & if not, could we remove it?

```
from mlflow.store.sqlalchemy_store import SqlAlchemyStore
```

seems to work for me

Reply:
I can remove it.

I added this to avoid importing sqlalchemy if I was not testing it since conftest loads this file and fails.